app/models/game.rb

```diff
@@ -2,4 +2,9 @@ class Game < ActiveRecord::Base
   has_many :matches
   has_many :users, through: :matches
   has_many :chess_pieces
+
+
+  def piece_in_square?(x,y)
+    chess_piece.where(position_x: x, position_y: y)
```

Comment:
so `where` will *always* return back an array like object, called `activerecord relation`. in `Ruby` only `false` and `nil` are considered `falsey` values.